app/views/layouts/base.html.erb

```diff
@@ -32,26 +32,26 @@
       <% end %>
     </script>
     <%= javascript_include_tag "locale/#{FastGettext.locale}/app" %>
-    <%= javascript_include_tag *webpack_asset_paths('foreman-vendor', :extension => 'js') %>
-    <%= javascript_include_tag *webpack_asset_paths('vendor', :extension => 'js') %>
-    <%= javascript_include_tag *webpack_asset_paths('bundle', :extension => 'js') %>
-    <%= javascript_include_tag 'application' %>
-    <%= webpacked_plugins_with_global_js %>
-    <%= webpack_dev_server %>
-    <%= yield(:javascripts) %>
-
-    <script type="text/javascript">
-      <%= yield(:inline_javascripts) %>
-    </script>
-
+    
     <%= yield(:head) %>
   </head>
 
   <body class="pf-m-redhat-font">
-    <!--[if lte IE 9]>
+    <!--[if IE]>
       <%= _('You are using an unsupported browser.') %>
+      <%= _('See list of supported browsers: %s') % 'https://theforeman.org/manuals/nightly/index.html#3.1.4BrowserCompatibility' %>
```

Comment:
We have a helper to link to documentation, which takes the version and branding into account.
```suggestion
      <%= _('See list of supported browsers: %s') % documentation_url('3.1.4BrowserCompatibility') %>
```

Reply:
true, that would be better, I will use it.